fastlane.gemspec

```diff
@@ -114,4 +114,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency('rest-client', '>= 1.8.0')
   spec.add_development_dependency('fakefs', '~> 0.8.1')
   spec.add_development_dependency('sinatra', '~> 1.4.8')
+  spec.add_development_dependency('xcov', '~> 1.4.1')
```

Comment:
Can we put a comment here to why we are requiring this as a dev dependency? (since its kind of an outlier) 😊